libraries/app/api.cpp

```diff
@@ -73,6 +77,87 @@ namespace graphene { namespace app {
        return true;
     }
 
+    bool login_api::login_signed( const string& b64_encoded_trx )
+    {
+      const string trx_json = fc::base64_decode( b64_encoded_trx );
```

Comment:
Not sure, but shouldn't there by some `try/catch` to ensure that any string (even not base64 encoded) can work and not cause the API to crash? Maybe add a unittest for random utf8 strings

Reply:
I will check this out. But I think I tried it with random strings and the parser throws an exception which is handled inside the application.